plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -1511,7 +1516,12 @@ public void registerCoreMembers() {
         // All the flag values are stored default in "plugins/denizen/saves.yml" file.
         // For an alternative way of storing values, use either yaml (See <@link command yaml>)
         // or sql (See <@link command sql>)
-        // TODO: Document Command Details
+        // Flags are not saved directly to file but stored in memory.
+        // It is recommended to use the save command (/denizen save) often and
+        // before the server shuts down if you want to keep the flags after restart.
+        // Duration of flags are not affected by server restarts and are still valid after a restart,
+        // assuming the flags are saved to file. Duration is not running on a timer,
+        // meaning it is possible for a duration to expire while the server is offline.

```

Comment:
This is an odd way of saying this, and either way doesn't really belong in the documentation of the flag command.

Reply:
The duration bit, though, should bear some mention of how the expiration occurs (on read timestamp comparison) if you're going to explain it in this detail.